src/hooks/useProfileModal.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback } from "react";
+
+import { userProfileSelector } from "utils/selectors";
+
+import { setUserProfileData } from "store/actions/UserProfile";
+
+import { useSelector } from "./useSelector";
+import { useDispatch } from "./useDispatch";
+import { User } from "types/User";
+import { WithId } from "utils/id";
```

Comment:
please, sort these according to `VenuePage.tsx`

Reply:
Roughly speaking the logic of the order goes from external/generic down to internal/specific; which roug equates to the following order:

- react
- other 3rd party libs
- [space]
- 'root imports' (eg. settings, etc)
- [space]
- imports from `types/`
- [space]
- imports from `utils/`
- [space]
- imports from `hooks/`
- [space]
- imports from `pages/`
- [optional space, sort of depends how many there are]
- imports from `components/templates`
- [optional space, sort of depends how many there are]
- imports from `components/organisms`
- [optional space, sort of depends how many there are]
- imports from `components/molecules`
- [optional space, sort of depends how many there are]
- imports from `components/atoms`
- [space]
- imports from `../` (if any.. and mostly only ever allowed to go up by '1 folder', but we always preference using the full import path for the above rather than going up 1 dir to it
- [space]
- imports from `./`
- [space]
- scss/other asset imports from `./`

This might not be 100% complete.. but that's generally how I go about doing it. (we have plans to enforce this with linting when I get the chance to implement it)